packages/server/src/utils.ts

```diff
@@ -107,19 +115,25 @@ export function normalizeNodeRequest(
    * because the presence of body means the request stream is already consumed and,
    * rawRequest cannot be used as BodyInit/ReadableStream by Fetch API in this case.
    */
-  const maybeParsedBody = nodeRequest.body;
+  const maybeParsedBody = nodeRequest.body || null;
```

Comment:
```suggestion
  const maybeParsedBody = nodeRequest.body || null;
```
Let's keep the changes as minimal as we can.